spark/src/main/scala/org/apache/comet/serde/QueryPlanSerde.scala

```diff
@@ -186,6 +186,13 @@ object QueryPlanSerde extends Logging with ShimQueryPlanSerde {
     }
   }
 
+  private def bitwiseAggTypeSupported(dt: DataType): Boolean = {
+    dt match {
+      case _: IntegerType | LongType | ShortType | ByteType => true
+      case _ => false
+    }
+  }
```

Comment:
Is the limitation due to DataFusion?

Reply:
If Spark bit aggregate function supports more types than DataFusion, can you add a comment here and open a ticket at DataFusion?

Reply:
Spark bitwise aggregates only supports `INTEGRAL` type. If I try other types, Spark throws Exception:
```
Cannot resolve "bit_and(col1)" due to data type mismatch: Parameter 1 requires the "INTEGRAL" type, however "col1" has the type ...
```